### DISCUSSION TITLE
GitHub Classroom assignment Download Grades has empty submission_timestamp

### DISCUSSION THREAD

**Discussion by hcientist** [community/community#147240] [GitHub Education]
### Hi everyone,

When I download the grades csv from my github classroom assignments, the `submission_timestamp` column is empty while in the web UI there are status indicators including `Submitted`, `Late`, `Not submitted`. Why is this column blank in the csv?

**Comment by eric-vader** [comment]
I encountered this also, its still blank

**Comment by nathanroberts72736-jpg** [comment]
This usually happens because the CSV export doesn’t include real-time submission metadata the same way the UI does, similar to how[ workday consultant roles and key skills ](https://dandeeconsulting.com/he-role-of-a-workday-consultant-skills-responsibilities-and-challenges/)are shown differently across reports versus dashboards depending on the data source.

**Reply by eric-vader** [reply]
Hmm do you know in what situations for which  it would return something rather than blank?
